waku/node/waku_switch.nim

```diff
@@ -101,7 +101,7 @@ proc newWakuSwitch*(
     if peerStoreCapacity.isSome():
       b = b.withPeerStore(peerStoreCapacity.get())
     else:
-      let defaultPeerStoreCapacity = int(round(float64(maxConnections)*1.25))
+      let defaultPeerStoreCapacity = int(maxConnections)*5
```

Comment:
Shall we have a comment indicating why we set the max PeerStoreCapacity to that?
From the PR's comment is clear, but I'd mention that in the code as well

Reply:
will address it in the future. maybe this shouldn't even be configurable and always harcoded depending on the amount of maxConnections.